internal/receiver/databricksreceiver/README.md

```diff
@@ -0,0 +1,33 @@
+# Databricks Receiver (Alpha)
+
+The Databricks Receiver uses the Databricks
+[API](https://docs.databricks.com/dev-tools/api/latest/index.html)
+to generate metrics about the operation of a Databricks instance.
+
+Supported pipeline types: `metrics`
+
+> :construction: This receiver is in **ALPHA**. Behavior, configuration fields, and metric data model are subject to change.
+
+## Configuration
+
+The following fields are required:
+
+- `instance_name`: A string representing the name of the instance. This value gets set as an `instance_name` resource attribute.
+- `base_url`: The protocol (http or https), hostname, and port for the Databricks API, without a trailing slash.
```

Comment:
There's been some effort in adopting [`HTTPClientSettings`](https://github.com/open-telemetry/opentelemetry-collector/blob/a7ef0790a0a59aae6f1cd1508b146f42ea8a24c0/config/confighttp/confighttp.go#L38) in different components, and it has a `ToClient()` method that can be used over the default.  This way `endpoint` would also be available for receiver creator instances to use.

Reply:
Done.